### DISCUSSION TITLE
Support for handshake domains ?

### DISCUSSION THREAD

**Discussion by C00K1E-dev** [WalletConnect/walletconnect-monorepo#4323] [Developer Support]
Hello 
I was wanting to verify a domain in walletconnect cloud , I made all the needed settings corect but it just wouldn't find my domain 
So I asked the domain provider and they said everything is set up correctly but I should ask here if handshake domains are supported.

**Comment by Cali93** [comment]
Hey @C00K1E-dev,
Could you provide us the domain you're trying to verify and your WalletConnect project ID ?

**Comment by C00K1E-dev** [comment]
Domain name :
https://starstrend.nfts

Project ID :
21467104516d58939fe60d2e0a5368e6

**Reply by Cali93** [reply]
It looks like there's a DNS resolution issue with your domain, which is not related to WalletConnect.
`curl --location 'https://starstrend.nfts'` returns `Could not resolve host: starstrend.nfts`.
Please first make sure your domain is configured correctly and then follow the instructions outlined in WalletConnect Cloud.
<img width="433" alt="Capture d’écran 2024-03-13 à 16 08 31" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/32299095/9f4b5d27-e63a-44b1-b2d4-86786f10aab5">

**Reply by C00K1E-dev** [reply]
well i just got off with support frome namecheap domain provider : this is the resolution  "I'm glad to inform you that the host record has been made successfully from my end. Be noted that it may take up to 30-60 minutes to propagate and accepted globally!" .... I hope it will work will test it again later

**Reply by C00K1E-dev** [reply]
@Cali93 Now when i try to verify the following message is being displayed : "Failed to fetch"

**Comment by C00K1E-dev** [comment]
@Cali93  your repository [scaffold-eth-w3m](https://github.com/Cali93/scaffold-eth-w3m) is intended to to replace all CLI of hardhat and make it user friendly ? IF so Great job man i will sure test that out today after work!

**Reply by Cali93** [reply]
It's just a fork of https://github.com/scaffold-eth/scaffold-eth-2 that uses Web3Modal. Scaffold-eth is a developer friendly Web3 starter kit.
